camayoc/tests/qpc/ui/utils.py

```diff
@@ -109,10 +132,12 @@ def create_credential(view, options):
                 options['credential_type'] + ' Credential')
     modal = CredentialModalView(view, locator=Locator(css='.modal-content'))
 
-    # workaround, should be `assert modal.save_button.disabled`
+    # Workaround, should be `assert modal.save_button.disabled`
     # https://github.com/RedHatQE/widgetastic.patternfly/pull/66
```

Comment:
can you file a camayoc issue to track this?

Reply:
done.